client/packages/system/src/Patient/ContactTrace/Components/columns.ts

```diff
@@ -11,55 +11,42 @@ import { ContactTraceRowFragment } from '@openmsupply-client/programs';
 export interface ContactTraceListColumnsProps {
   onChangeSortBy: (column: Column<ContactTraceRowFragment>) => void;
   sortBy: SortBy<ContactTraceRowFragment>;
-  includePatient?: boolean;
 }
 
 export const useContactTraceListColumns = ({
   onChangeSortBy,
   sortBy,
-  includePatient,
 }: ContactTraceListColumnsProps) => {
   const { localisedDate } = useFormatDateTime();
 
-  const columnList: ColumnDescription<ContactTraceRowFragment>[] = [];
-if (includePatient) {
-    columnList.push({
-      key: 'patientName',
-      label: 'label.patient',
-      accessor: ({ rowData }) => rowData?.patient?.name,
-      sortable: false,
-    });
-  }
-  columnList.push(
+  const columnList: ColumnDescription<ContactTraceRowFragment>[] = [
     {
       key: 'programName',
       label: 'label.program',
       accessor: ({ rowData }) => rowData.program.name,
       sortable: false,
     },
     {
-      key: 'contactTraceId',
-      label: 'label.contact_trace-id',
+      key: 'datetime',
+      label: 'label.date',
```

Comment:
the label in the issue is `Created/Added on` (which I think is better as `Date created` or `Created date` anyway)